src/Java/Widgets/SeedWidget.md

```diff
@@ -0,0 +1,5 @@
+### Description
+
+This example demonstrates how to use vtkSeedWidget, which generates (seeds) points to be placed in the scene in the locations where the user clicks.
+
+The points can then be used for operations like connectivity, segmentation, and region growing. For an example using a custom callback where such operations can be assembled, see SeedWidgetWithCustomCallback.
```

Comment:
Regarding SeedWidgetWithCustomCallback are you referring to the Cxx example as the Java example does not exist?
If so this needs to be a link, look at the markdown file in e.g. [Cube.cxx](https://lorensen.github.io/VTKExamples/site/Cxx/GeometricObjects/Cube/) for how to do this.